### DISCUSSION TITLE
wrappers的代码在哪里可以查看？

### DISCUSSION THREAD

**Discussion by rjl-cli** [umijs/umi#12470] [Ideas]
wrappers的代码在哪里可以查看？
umi非常好，但是，对于一些特殊的组件库，如react-scroll-parallax
使用是非常不便的。
最好的是，引入umi的wrappers 的逻辑代码。但是我并没有找到，能否告诉我在哪里？

**Comment by fz6m** [comment]
See https://github.com/umijs/umi/issues/12200#issuecomment-2183562101

如有更多信息，需提供一个最小复现，或更详细的上下文描述。
